### DISCUSSION TITLE
Does not getting proper response

### DISCUSSION THREAD

**Discussion by kannan-egrove** [deepgram/community#197] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

We are sending the video to  deepgram transcribe API.But the generated  transcribe is differnet from actual video content.

**Comment by jjmaldonis** [comment]
Hi @kannan-egrove can you please provide additional context?

How is the transcript different than what you expect? Are there specific words that are different?

What language is the audio in?

Is the `confidence` of the words that are different very low?

What is the video file format? Is it being sent to Deepgram via a URL or from a local file on the computer/server?

What is the URL (including query parameters) you are using?

If you can include the audio file and the code that will help us help you.

**Reply by kannan-egrove** [reply]
Hi @jjmaldonis thanks for replay

https://api.deepgram.com/v1/listen?detect_language=true&punctuate=true&diarize=true&channels=2 i am using this url

my video format is mp4. there specific words that are different.

https://class2learnnew.s3.amazonaws.com/deepgram_video/1686125229video_1007.mp4

we are getting different transcribed data output
